pkgs/applications/misc/khard/default.nix

```
@@ -15,6 +17,7 @@ python3.pkgs.buildPythonApplication rec {
     sphinxHook
```

```suggestion
    setuptools
    sphinxHook
```
We set `pyproject = true` (`format = "pyproject"`), so setuptools is no longer included in nativeBuildInputs.
Could you please add it explicitly?
https://github.com/lucc/khard/blob/8d841537b5cedc0817adcd49560978f3a546deb1/pyproject.toml#L46